### PR TITLE
Add vue-pwa-vision & vue-pwa-rekognition

### PR DESCRIPTION
**Pull request by aofdev**
Add vue-pwa-vision & vue-pwa-rekognition
**Category:** Examples
**Github Link:**
https://github.com/aofdev/vue-pwa-vision
https://github.com/aofdev/vue-pwa-rekognition

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -347,6 +347,8 @@ vue-router 2.0, vue-infinite-scroll 2.0, vue-progressbar 2.0 by [TIGERB](https:/
   - [Vue Weather Notifier](https://github.com/sdras/vue-weather-notifier) A small sample animation app with SVG and Vuex
   - [Nuxt Type](https://github.com/sdras/nuxt-type) An example Vue project with Nuxt for routing/SSR to demo page transitions
   - [VueBlog](https://github.com/wmui/vueblog) A blog system supporting service side rendering by [wmui](https://github.com/wmui)
+  - [vue-pwa-vision](https://github.com/aofdev/vue-pwa-vision) - Vue + Vuetify + Google Cloud Vision Face Detection With Progressive Web App by [aofdev](https://github.com/aofdev)
+- [vue-pwa-rekognition](https://github.com/aofdev/vue-pwa-rekognition) - Vue + Vuetify + Amazon Rekognition Face Detection with Progressive Web App by [aofdev](https://github.com/aofdev)
```
Comment (kazupon) [inline README.md]: please adjust indent. 🙏

**Comment by aofdev** [comment]
update adjust indent

File: README.md [hunk 2]
```diff
@@ -424,6 +426,7 @@ vue-router 2.0, vue-infinite-scroll 2.0, vue-progressbar 2.0 by [TIGERB](https:/
 - [CoreUI](https://github.com/mrholek/CoreUI-Free-Bootstrap-Admin-Template) - Open Source Admin Template powered by Vue.js
 - [ChuckNorris](https://github.com/mazipan/chucknorris) - Chuck Norris Jokes Generator built with VueJS + api.chucknorris.io
 
+
```
Comment (kazupon) [inline README.md]: why did you add a blank line? 🤔

**Comment by aofdev** [comment]
i am sorry

**Comment by aofdev** [comment]
thanks you
